src/images/tpc.cpp

```diff
@@ -100,6 +100,18 @@ void TPC::readHeader(Common::SeekableReadStream &tpc, byte &encoding) {
 	_layerCount = 2;
 	_isAnimated = checkAnimated(width, height, dataSize);
 
+	if (_isAnimated) {
```

Comment:
So animated TPC don't correctly specify the mipmap count in the header, so you need to count them?

Reply:
Yes, exactly. My work here is based on tpc2tga which is doing the exact same thing. Don't ask me what was the intention of creating down to 1x1 mipmaps. But this concept fit perfectly in the remaining size.